src/entities/User.ts

```diff
@@ -42,19 +42,31 @@ class User {
   graduationYear: string | null;
 
   /** User's major */
-  @ManyToOne(type => CornellMajor, major => major.users, { nullable: true, onDelete: "CASCADE" })
+  @ManyToOne(
+    type => CornellMajor,
+    major => major.users,
+    { nullable: true, onDelete: 'CASCADE' }
+  )
```

Comment:
nice, we should use a repo wide code formatter like prettier if we aren't already

Reply:
Yeah, I'm working on a pre-commit hook that will run prettier/tslint automatically for us.